eng/tools/Directory.Build.props

```diff
@@ -5,5 +5,6 @@
     <IsPackable>false</IsPackable>
     <DisablePackageReferenceRestrictions>true</DisablePackageReferenceRestrictions>
     <Nullable>disable</Nullable>
+    <DisableImplicitNamespaceImports>true</DisableImplicitNamespaceImports>
```

Comment:
Why not just web?

Reply:
The tools use Microsoft.Build.Tasks which conflicts with System.Threading.Tasks. Since it's just tools, I just disabled everything instead of just the specific conflicts.

Reply:
Why does it conflict? Because we don't have the new compiler that hides these conflict warnings? Can you clarify?

Reply:
It's not a warning, it's an error for ambiguous types.

`error CS0104: 'Task' is an ambiguous reference between 'Microsoft.Build.Utilities.Task' and 'System.Threading.Tasks.Task`